use-cases.md

```diff
@@ -0,0 +1,72 @@
+# CloudEvents - Use Cases
+
+[WIP] Use-case examples to help end users understand the value of OpenEventing.
+
+### Normalizing Events Across Services & Platforms
+
+Major event publishers (e.g. AWS, Microsoft, Google, etc.) all publish events
+in different formats on their respective platforms.  There are even a few cases
+where services on the same provider publish events in different formats (e.g.
+AWS).  This forces event consumers to implement custom logic to read or munge
+event data across platforms and occasionally across services on a single
+platform.
+
+OpenEvents can offer a single experience for authoring consumers that handle
+events across all platforms and services.
+
+### Facilitating Integrations Across Services & Platforms
+
+Event data being transported across environments is increasingly common.  
+However, without a common way of describing events, delivery of events across
+environments is hindered.  There is no single way of determining where an event
+came from and where it might be going.  This prevents tooling to facilitate
+successful event delivery and consumers from knowing what to do with event
+data.
+
+OpenEvents offers useful metadata which middleware and consumers can rely upon
+to facilitate successful event delivery and receipt.
+
+### Increasing Portability of Functions-as-a-Service
+
+Functions-as-a-Service (also known as serverless computing) is one of the
+fastest growing trends in IT and it is largely event-driven.  However, a
+primary concern of FaaS is vendor lock-in.  This lock-in is partially caused
+by differences in function APIs and signatures across providers, but the
+lock-in is also caused by differences in the format of event data received
+within functions.
+
+OpenEvents' common way of describing event data increases the portability of
+Functions-as-a-Service.
+
+### Improving Development & Testing of Event-Driven/Serverless Architectures
```

Comment:
Today, if I am locked into a single platform, I can write a few lines of code to mock the events specific to that platform. However, no one is investing in building tools/libraries that would let me do this efficiently. So the highlight of this use case needs to be the boost that this spec will give to the development of tools that help with the end-to-end lifecycle of serverless architecture.

Reply:
Yep.  I've modified the language slightly for clarity.  Though, I believe the language already communicates your comment fairly well.  Open to suggestions...